squad/frontend/templates/squad/base.html

```diff
@@ -73,7 +74,6 @@
     <script type="text/javascript" src="{% static "lodash.js" %}"></script>
     <script type="text/javascript" src="{% static "chartjs/Chart.bundle.js" %}"></script>
     <script type="text/javascript" src="{% static "floatThread/dist/jquery.floatThead.js" %}"></script>
-    <script type="text/javascript" src='{% static "squad/table.js" %}'></script>
```

Comment:
don't we want this for all tables? (at least in the past we did)

Reply:
It's very annoying for the table in the 'test compare' page. The header becomes floating and the table is pretty unusable.